lib/utils/checkUpdatedPackages.js

```diff
@@ -20,11 +26,14 @@ module.exports = function checkUpdatedPackages(packagesLoc) {
 
   var hasTags = gitUtils.hasTags();
   var lastTagCommit;
-  var lastTag;
+  var commits;
 
-  if (hasTags) {
+  if (flags.canary) {
+    var currentSHA = gitUtils.getCurrentSHA();
```

Comment:
Instead of the `getCurrentSHA`, we could change this to be a default and either prompt the user or even better just use a passed in sha from the flag (flag.canary). Would be cool to support just passing in a PR number too